README.md

```diff
@@ -5,6 +5,27 @@ the
 user like authenticating the users, issuing the access/refresh tokens, api keys, validating the permissions for the

```

Comment:
this API is responsible **for** any service related to ... of the user, like , aut